### PR TITLE
HTS-1097 made nino optional

### PR DESCRIPTION
**Pull request by Jackie1988**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavetestadminfrontend/connectors/AuthConnector.scala [hunk 1]
```diff
@@ -29,25 +29,28 @@ import scala.concurrent.{ExecutionContext, Future}
...
 
-  def getRequestBody(nino: String): String =
+  def getRequestBody(nino: Option[String]): String =
     s"""{
          "authorityId":"htsapi",
          "affinityGroup":"Individual",
          "confidenceLevel":200,
          "credentialStrength":"strong",
-         "nino":"$nino",
+         "nino":"${nino.fold("")(_.toString)}",
```
Comment (Jackie1988) [inline app/uk/gov/hmrc/helptosavetestadminfrontend/connectors/AuthConnector.scala]: should be:
```scala
  "nino" : ${nino.fold("null"){n => """"n""""}}
```

Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosavetestadminfrontend/connectors/AuthConnector.scala]: or:
```scala
 "nino" : ${nino.fold("null"){n => "\"n\""}}
```

**Comment by sureshhmrc** [comment]
whats the benefit of making nino ? i mean you are making these changes at all ?  
if nino is not present then it would be difficult to retrieve the bearer token

**Comment by Jackie1988** [comment]
We have changed how the token is generated

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**